capstone/src/main/java/com/google/step/data/Restaurant.java

```diff
@@ -160,7 +160,6 @@ public int hashCode() {
     result = prime * result + ((location == null) ? 0 : location.hashCode());
     result = prime * result + ((name == null) ? 0 : name.hashCode());
     result = prime * result + ((phone == null) ? 0 : phone.hashCode());
-    result = prime * result + (int) (restaurantKey ^ (restaurantKey >>> 32));
```

Comment:
Have a look at (I think) Josh's changes. He's using a library that makes these a lot more manageable.